third_party/qpg_sdk/make.py

```diff
@@ -0,0 +1,7 @@
+#! /usr/bin/env python3
```

Comment:
since this is in the chip repo, could we have the chip license blurb added (even if content is somewhat trivial)?

Reply:
I added the license notice